documents/purchaseorder/model_test.go

```diff
@@ -44,6 +44,7 @@ func TestMain(m *testing.M) {
 		&testlogging.TestLoggingBootstrapper{},
 		&config.Bootstrapper{},
 		&storage.Bootstrapper{},
+		&identity.Bootstrapper{},
```

Comment:
Can we remove pointer receiver for this bootstrap?

Reply:
I default to pointer because that saves the overhead of copying for any purpose

Reply:
Pointers create overhead on Garbage collection since this will go on the heap instead of stack. It is recommended to avoid as much as we can unless the struct is so big that copying becomes an overhead.